client/my-sites/marketing/tools/style.scss

```diff
@@ -1,3 +1,51 @@
+.tools__header-body {
+	display: flex;
+	flex-direction: row;
+	padding: 11px 24px; // standard padding
+
+	@include breakpoint( '>1040px' ) {
+		max-height: 300px;
+	}
+}
+
+.tools__header-info {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.tools__header-title {
+	font-size: 21px;
+	font-weight: 400;
+	margin: 10px 5px;
```

Comment:
We should remove left and right margins as it causes an alignment issue:

![screenshot](https://user-images.githubusercontent.com/594356/56913375-c6e28380-6ab1-11e9-9365-0bd8b0d193ca.png)